scripts/export.sh

```diff
@@ -30,19 +31,34 @@ nohup lily daemon --repo="${REPO_PATH}" --config=/lily/config.toml --bootstrap=f
 # Wait for Lily to come online
 lily wait-api
 
-# Extract the walking epochs
-FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+# Extract the available walking epochs
+STATE=$(lily chain state-inspect -l 3000)
+# FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+FROM_EPOCH=$(echo "${STATE}" | jq -r ".summary.stateroots.oldest")
+FROM_EPOCH=$((FROM_EPOCH + 2))
+# Add WALKEPOCHS to the FROM_EPOCH
 TO_EPOCH=$((FROM_EPOCH + WALK_EPOCHS))
+# TO_EPOCH=$(echo "${STATE}" | jq -r ".summary.stateroots.newest")
 
 echo "Walking from epoch ${FROM_EPOCH} to ${TO_EPOCH}"
 sleep 10
 
 # Run export
-# archiver run --storage-path /tmp/data --ship-path "${EXPORT_DIR}" --min-height="${FROM_EPOCH}" --max-height="${TO_EPOCH}"
+# archiver export --storage-path /tmp/data --ship-path "${EXPORT_DIR}" --min-height="${FROM_EPOCH}" --max-height="${TO_EPOCH}"
```

Comment:
Once we validate that the `export` command is working properly, we can basically comment out/delete the rest of the script!